assets/javascripts/discourse/services/ai-conversations-sidebar-manager.js

```diff
@@ -26,6 +27,7 @@ export default class AiConversationsSidebarManager extends Service {
 
     this.sidebarState.isForcingSidebar = true;
     document.body.classList.add("has-ai-conversations-sidebar");
+    this.appEvents.trigger("discourse-ai:force-custom-sidebar");
```

Comment:
Could you change the name of this event to be clear that it's for the bot_conversations sidebar? Maybe `discourse-ai:force-conversations-sidebar`?